doc/schemas/app_openapi.json

```diff
@@ -622,11 +618,9 @@
           {
             "in": "query",
             "name": "id",
-            "required": false,
+            "required": true,
```

Comment:
It was a bug in the previous API generation that marked all of these query params as `"required": false` - most would actually throw a JSON parsing error if the field wasn't present. Now they're correctly marked as required, and the error if they're missing more clearly refers to the query string.